client/src/app/site/motions/services/motion-repository.service.ts

```diff
@@ -56,24 +56,57 @@ export class MotionRepositoryService extends BaseRepository<ViewMotion, Motion>
     }
 
     /**
-     * Creates and updates a motion
+     * Creates a motion
+     * Creates a (real) motion with patched data and delegate it
+     * to the {@link DataSendService}
+     *
+     * @param update the form data containing the update values
+     * @param viewMotion The View Motion. If not present, a new motion will be created
+     */
+    public create(update: any, viewMotion?: ViewMotion): Observable<any> {
+        return this.save(update, viewMotion);
+    }
+
+    /**
+     * updates a motion
      *
      * Creates a (real) motion with patched data and delegate it
      * to the {@link DataSendService}
      *
      * @param update the form data containing the update values
      * @param viewMotion The View Motion. If not present, a new motion will be created
      */
-    public saveMotion(update: any, viewMotion?: ViewMotion): Observable<any> {
+    public save(update: any, viewMotion?: ViewMotion): Observable<any> {
         let updateMotion: Motion;
-
         if (viewMotion) {
             // implies that an existing motion was updated
             updateMotion = viewMotion.motion;
         } else {
             // implies that a new motion was created
             updateMotion = new Motion();
         }
+        // submitters: User[] -> submitter: MotionSubmitter[]
+        const submitterArr = update.submitters as User[];
```

Comment:
Please use `submitters` and `newSubmitterIds`. The same for supporters and every other array ;)